app/actions/remote/post.ts

```diff
@@ -357,7 +365,7 @@ export async function fetchPosts(serverUrl: string, channelId: string, page = 0,
         if (!fetchOnly) {
             const models = await operator.handlePosts({
                 ...result,
-                actionType: ActionType.POSTS.RECEIVED_SINCE,
+                actionType: ActionType.POSTS.RECEIVED_IN_CHANNEL,
```

Comment:
Fetch Posts is supposed to get the latest posts, but not necessarily since the last chunk we have, so better handle it as `RECEIVED_IN_CHANNEL` and let the chunk merge deal with any fragmentation.